uc-crux-llvm/src/UCCrux/LLVM/Context/Function.hs

```diff
@@ -205,21 +208,23 @@ tryMakeFunctionContext modCtx defnSymb argFullTypes argTypes =
           _argumentNames =
             fmapFC
               (Const . getFirst . getConst)
-              ( mapToContext
+              ( intMapToContext
                   (Ctx.size argFullTypes)
-                  (fmap (First . Just) (debugInfoArgNames (getModule llvmMod) def))
+                  (fmap
+                     (First . Just . Text.pack)
+                     (debugInfoArgNames (getModule llvmMod) def))
               )
         }
 
-mapToContext ::
+intMapToContext ::
```

Comment:
Do the callers care if it's a Map or an IntMap?  If not, keeping the original generalized name helps immunize the callers against needing to know this detail.

Reply:
Presumably, in that they need to pass in differently-typed arguments. There's also only one caller, and it's just above (this function is not exported).